### DISCUSSION TITLE
Microsoft Edge

### DISCUSSION THREAD

**Discussion by zaid-lol** [memstechtips/UnattendedWinstall#206] [Ideas]
Can you modify this part of the registry so that Microsoft Edge can be uninstalled? this is it "Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Windows\CurrentVersion\Uninstall\Microsoft Edge"
you just have to change the "NoRemove" value to 0 instead of one. Now I have no idea if that is even possible I don't know much about answer files but I'm giving a suggestion, so if it is possible can you please add it?

Looking forward to your reply.
![Screenshot 2024-12-23 134129](https://github.com/user-attachments/assets/1338960e-c33e-4819-8d17-98753b58b6f7)

**Comment by memstechtips** [comment]
Hey there and sorry for the late reply,

Since the creation of this issue I have released a new version of the XML file and a major update to [Winhance](https://winhance.net/), I recommend that you watch [this video](https://youtu.be/lrq3ph3xi50) which provides a full demo of the current version along with how to use the new version of WIMUtil and how to install the drivers etc. You now also have the ability to create an autounattend.xml file based on your selections in Winhance, so you have more control over which apps are removed and which settings are applied.

I have also created [this video](https://youtu.be/sCd1TI6AbOI) about v3.0.0 of UnattendedWinstall, I suggest that you watch it too if you're interested.

There is full Microsoft Edge uninstallation in Winhance.

I'm closing this issue now, but if you need more help with the same issue, feel free to reopen it.
